Gek62/hometask-4/my_array_rotate_benchmark.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'benchmark'
+require './my_array_rotate'
+
+Benchmark.bm do |x|
```

Comment:
Please, make sure that you test all significant scenarios, like
- rotate by some small number (you've already done it)
- rotate by a number that is equal to array size
- rotate by a number that is more than the array's size
- rotate by a negative number